src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -113,20 +112,60 @@ export const JiraConfigurationGet = async (
 				isGlobalInstall: installation.repository_selection === "all"
 			}));
 
+		/**
```

Comment:
I don't think we should leave this huge TODO in here. Better to add details to [ARC 1471](https://softwareteams.atlassian.net/jira/software/projects/ARC/boards/385/backlog?selectedIssue=ARC-1471&text=jira-con) to outline how we need to munge the data from the backend so it renders correctly.

Reply:
This has been removed.
But, I add added here to provide a data sample, for anyone who wants to test the UI. What would be the best place to add such data?

Reply:
I would create a ticket for `TODO: fetch the actual list of GHE servers` and add it to that. The auto/manual app creation isn't implemented yet but we can `GitHubServerApp.install(payload)` to create entries in the db. This would allow us to implement rendering data from the database instead of mocking it out and we wouldn't need to save the data blob you had here anywhere as we'll effectively setting it up to take the data and programmatically render it.

Reply:
I've created a new ticket for this. https://softwareteams.atlassian.net/browse/ARC-1488. Once the new endpoint is done, this part can be then be updated to fetch the actual list.

Reply:
That ticket doesn't need to wait on the endpoint. We already have methods that exist in the GHSA app model to install and retrieve data from that table.

Reply:
Updated the blank array to fetch actual GH Server apps data